addons/binding/org.openhab.binding.zwave/ESH-INF/thing/_channels.xml

```diff
@@ -230,6 +230,16 @@
                 </options>
                 <limitToOptions>true</limitToOptions>
             </parameter>
+            <parameter name="config_invert_percent" type="boolean">
+                <label>Invert dimmer value</label>
+                <description>Invert the blinds dimmer value</description>
```

Comment:
Sure. I was unsure what to name it, and just selected dimmer. I will upload a patch.